patches/@rushstack+eslint-patch+1.1.4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@rushstack/eslint-patch/lib/modern-module-resolution.js b/node_modules/@rushstack/eslint-patch/lib/modern-module-resolution.js
```

Comment:
I think this is out of scope for this PR. We should create a new PR in the `prettier-eslint` package that addresses the @rushstack/eslint-patch issue reported in #434

Reply:
Without this patch, the test will fail, `@rushstack/eslint-patch` is out controlled, and it is only a nested dev dependency via `eslint-config-kentcdodds`, so I don't think we should exclude this part.

Reply:
@JounQin Gotcha. I didn't know it was a dependency of `eslint-config-kentcdodds`